.github/workflows/build-wheel-macos-x86_64.yaml

```diff
@@ -344,13 +341,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [macos-12]
-        arch: [x86_64]
         python_version: ${{ fromJson(needs.constants.outputs.python_versions) }}
 
     # To check all wheels for supported python3 versions
-    name: Test Wheels (Python ${{ matrix.python_version }}) on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
+    name: Test Wheels (Python ${{ matrix.python_version }}) on Mac x86
+    runs-on: macos-12
```

Comment:
is it not runner.os?

Reply:
So `runner.os` is a built-in variable that specifies the OS, which is not the same as a runner label. The runner labels determine the os (version), architecture, physical resources of the machine, etc. See here for the list:
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

I don't think `runner.os` would even be available before you specified the runner.

Reply:
Oh I did not know, thanks :+1: